datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -175,6 +176,10 @@ export class DatasetEntity implements Entity<Dataset> {
                         },
                     },
                 },
+                {
+                    name: 'Access Management',
+                    component: AccessManagement,
```

Comment:
Hi @jjoyce0510 

Thanks for the comment. Before further changes
For confirmation i have queries could you please help regarding same.
1. For the tabs we already have isenabled as well as is visible condition based on the data values. Can we use the same to make the Access management tab disabled?
2. Could you please help regarding tabs enabled and visible properties that we can see in datasetentity.tsx  also which which property we can use as per the requirement?

Thanks
Ramendra

Reply:
Hi! I'm really interested on the access management topic so glad to see there is some activity on this topic!

However, this comment here really concerned me 

> Otherwise, all users wil start to see this and it will not be applicable to most!

If this is not applicable to most of DataHub users how is all this ends up in the public version? Specially considering that the PDL models and GraphQL mappers were unconditionally included. What's the long term plan for this feature?

Reply:
I think the comment only meant that it is not enabled by default. Admins can still choose to enable it. Like the new browse experience is not yet enabled by default but admins can control that via the env variables.